editor/plugins/tile_map_editor_plugin.cpp

```diff
@@ -1942,9 +1957,14 @@ TileMapEditorPlugin::TileMapEditorPlugin(EditorNode *p_node) {
 	EDITOR_DEF("editors/tile_map/sort_tiles_by_name", true);
 	EDITOR_DEF("editors/tile_map/bucket_fill_preview", true);
 	EDITOR_DEF("editors/tile_map/show_tile_info_on_hover", true);
+	EDITOR_DEF("editors/tile_map/editor_side", 1);
+	EditorSettings::get_singleton()->add_property_hint(PropertyInfo(Variant::INT, "editors/tile_map/editor_side", PROPERTY_HINT_ENUM, "Left,Right"));
 
 	tile_map_editor = memnew(TileMapEditor(p_node));
-	add_control_to_container(CONTAINER_CANVAS_EDITOR_SIDE, tile_map_editor);
+	if (EditorSettings::get_singleton()->get("editors/tile_map/editor_side"))
```

Comment:
Same comment here - if you find the switch case too much I'm ok with the `if`, but then I'd make it
`if (EditorSettings::get_singleton()->get("editors/tile_map/editor_side") == 1) { // Right`

Whichever you prefer, should be made consistent with the style used in `_notification`.

Reply:
Sorry 'bout that, completely forgot about that section. I'm fine with using `switch`.